src/commands/case_management.py

```diff
@@ -0,0 +1,590 @@
+
+    if not rescue:
+        await ctx.reply("No case with that name or number.")
+        return
+
+    async with ctx.bot.board.modify_rescue(rescue) as case:
+        case.active = False
+        case.status = Status.CLOSED
+        if first_limpet:
+            case.first_limpet = first_limpet
+            # TODO: Add paperwork call link here
+
+    # FIXME: Deleting case from the board, as we don't have a proper method yet.
+    del ctx.bot.board[rescue.board_index]
+
+    await ctx.reply(f"Case {case.client} was cleared!")
+
+
+@require_channel
+@require_permission(RAT)
+@command("cmdr", "commander")
+async def cmd_case_management_cmdr(ctx: Context):
+    if len(ctx.words) < 2:
+        await ctx.reply("Usage: !cmdr <Client Name|Board Index> <CMDR name>")
+        return
+
+    # Pass case to validator, return a case if found or None
+    rescue = _validate(ctx, ctx.words[1])
+
+    if not rescue:
+        await ctx.reply("No case with that name or number.")
+        return
+
+    async with ctx.bot.board.modify_rescue(rescue) as case:
+        case.client = ctx.words[2]
+        await ctx.reply(f"Client for {case.board_index} is now CMDR {case.client}")
+
+
+@require_channel
+@require_channel(RAT)
+@command("codered", "casered", "cr")
+async def cmd_case_management_codered(ctx: Context):
+    if len(ctx.words) < 2:
+        await ctx.reply("Usage: !codered <Client Name|Board Index>")
+        return
+
+    # Pass case to validator, return a case if found or None
+    rescue = _validate(ctx, ctx.words[1])
+
+    if not rescue:
+        await ctx.reply("No case with that name or number.")
+        return
+
+    async with ctx.bot.board.modify_rescue(rescue) as case:
+        case.code_red = not case.code_red
+        if case.code_red:
+            await ctx.reply(f"Code Red! {case.client} is on Emergency Oxygen!")
+        else:
+            await ctx.reply(f"{case.client} is no longer a Code Red.")
+
+
+@require_channel
+@require_permission(OVERSEER)
+@command("delete")
+async def cmd_case_management_delete(ctx: Context):
+    if len(ctx.words) < 2:
+        await ctx.reply("Usage: !delete <Database ID>")
+        return
+
+    # Validate we have UUID:
+    try:
+        requested_case = uuid.UUID(ctx.words[1])
+    except ValueError:
+        await ctx.reply("Invalid Database ID.")
+        return
+    else:
+        rescue = ctx.bot.board.get(requested_case)
+        # TODO: if the case is not present on the board, present the request to the API
+        #  and delete that case.
+
+        await ctx.reply(f"Deleted case with id {str(rescue.api_id)} - THIS IS NOT REVERTIBLE!")
+        # FIXME: Add proper method to delete a case from the board
+        del ctx.bot.board[requested_case]
+
+
+@require_channel
+@require_permission(RAT)
+@command("epic")
+async def cmd_case_management_epic(ctx: Context):
+    # This command may be depreciated, and not used.  It's left in only as an artifact, or
+    # if that changes.
```

Comment:
[suggestion] emit a deprication warning 😉 
```suggestion
    # if that changes.
    warnings.warn("This command is a deprecated artefact", DeprecationWarning)
```

Reply:
I'm not so sure a log message warning of a deprecated command is very useful.. We've already got the IRC reply, what does logging do for those who look at the logs?

Reply:
tells whoever is managing the logs that a depricated command was called, which is probably a problem.